src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -163,7 +163,7 @@ export class FollowArtistButton extends React.Component<Props, State> {
 
     // Custom button renderer
     const content = render ? (
```

Comment:
Will need to follow up on where `render` is invoked throughout codebase. So something like 
```js
<FollowArtistButton render={...}
```
This `<Clickable>` change seems to only cover the default use-case.

Reply:
Goood call!